heet-2312/README.md

```diff
@@ -0,0 +1,33 @@
+# 100DaysOfCode
+
+***
+## Heet Surana
+### Second year CSE,MNIT Jaipur
+-----------------------------------------------------------------------------------------
+
+## Goals
+-----------------------------------------------------------------------------------------
+### Competitive Programming
+ + Increase CodeChef rating to 1800
+ + Increase CodeForces rating to 1500
+ + Solve 50 practice questions on codechef,codeforces,spoj.
+
+
+### Learning Goals
+ + Remaining backend part-mongodb,authentication,etc.
+ + Learn Linear Regression,Logistic Regression and K-NN algorithm
+ + Learn Basic React(last priority)
+ + If possible learn Web Scrapping with Python using Scrapy
+
+### Projects
+ + One Web-Development Project
+ + One Machine Learning Project
+------------------------------------------------------------------------------------------
+## Timeline
+
+## Day 1(January 5,2020)
+  +Solved 1 problem on codeforces
+  +learned express and working with API's
+	|Problems solved    |
+	|----------------   |
+	|[I'm an inline-style link with title](https://codeforces.com/contest/1287/problem/A "Angry Students(Codeforces)")|
```

Comment:
You need to change `I'm an inline-style link with title` with the title of the link :)

Reply:
Also please see if you can resolve the existing conflicts. You will need to make a PR from codeshows to your branch

Reply:
so do i have to go into the codeshows repository and create a pull request to my branch?

Reply:
You can also create it by going to your repo :)
Both work